external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/PartitionManager.java

```diff
@@ -41,29 +41,29 @@ public PartitionManager(
 
     super(spoutConfig, partitionId, stateStore, receiver);
     
-    this.pending = new LinkedHashMap<String, EventData>();
-    this.toResend = new TreeSet<EventData>();
+    this.pending = new LinkedHashMap<String, EventDataWrap>();
+    this.toResend = new TreeSet<EventDataWrap>();
```

Comment:
it is indeed ordered by sequence number. The compareto definition in eventdatawrap class is as follows:
@Override
  public int compareTo(EventDataWrap ed) {
    return messageId.getSequenceNumber().
        compareTo(ed.getMessageId().getSequenceNumber());
  }